lego/apps/events/serializers/events.py

```diff
@@ -111,29 +120,38 @@ class EventCreateAndUpdateSerializer(TagSerializerMixin, BasisModelSerializer):
     cover = ImageField(required=False, options={'height': 500})
     pools = PoolCreateAndUpdateSerializer(many=True, required=False)
     text = ContentSerializerField()
+    is_abakom_only = BooleanField(required=False, default=False)
 
     class Meta:
         model = Event
         fields = (
             'id', 'title', 'cover', 'description', 'text', 'company', 'feedback_description',
             'feedback_required', 'event_type', 'location', 'is_priced', 'price_member',
             'price_guest', 'use_stripe', 'payment_due_date', 'start_time', 'end_time', 'merge_time',
-            'use_captcha', 'tags', 'pools', 'unregistration_deadline', 'pinned', 'heed_penalties'
+            'use_captcha', 'tags', 'pools', 'unregistration_deadline', 'pinned', 'heed_penalties',
+            'is_abakom_only'
         )
 
     def create(self, validated_data):
         pools = validated_data.pop('pools', [])
+        is_abakom_only = validated_data.pop('is_abakom_only', False)
         with transaction.atomic():
             event = super().create(validated_data)
             for pool in pools:
                 permission_groups = pool.pop('permission_groups')
                 created_pool = Pool.objects.create(event=event, **pool)
                 created_pool.permission_groups.set(permission_groups)
-
+            if is_abakom_only:
+                event.require_auth = True
```

Comment:
Would be nice to have this as a model method. Then you can just call e.g. `event.set_abakom_only()` both here and in update